MinnetonkaGame.py

```diff
@@ -41,12 +43,44 @@ def big_room():
         game_over("They ask you to show your skill by timing you on a Minecraft speed run. " +
                   "You fail miserably as you have never played before.")
     elif answer == "2":
-        # TODO: top_of_stairs_landing()
+        top_of_stairs_landing()
         game_over("TBD")
     else:
         big_room()
 
 
+# top of stairs
+def top_of_stairs_landing():
+    # give some prompts
+    # '\n' is to print the line in a new line
+    print("\nYou are in a living room. There is a woman speaking French. Did you teleport to France?")
+    # pause for two seconds - building suspense
+    time.sleep(2)
+    print("No. That's just Emily - an American who speaks French.")
+    time.sleep(2)
+    print("You walk over to say hi and you step on a lego. Ow!")
+    time.sleep(2)
+    print("Your foot is killing you! Do you take an aspirin or have a beer?")
+    print("1) Take an aspirin.")
+    print("2) Dull the pain with beer.")
+
+    # take input()
+    answer = input(">")
+
+    if answer == "1":
+        game_over("That aspirin was tainted with cyanide. Too bad. ")
+    elif answer == "2":
+        fridge()
+    else:
+        # else call game_over() function with the "reason" argument
+        game_over("Don't you know how to type a number?")
+
+
+# at fridge
+def fridge():
+    pass
```

Comment:
I'm not familiar with "pass". Is that just sort of a Python place holder for "pretend there's some code here but it does nothing"?

Reply:
yes, exactly. We get an unresolved reference error if we call fridge and it's not defined yet.  It's used a lot for functions like I had it or classes you know you're going to need later but don't feel like defining yet. To avoid an error you'd just say: 

```
class foo:
   pass
```